src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+       action="/search"
+       method="GET"
+       data-qa="big"
+       class="search-bar"
+       >
+      <img
+         src="./images/Search.svg"
+         alt="Search icon"
+         class="search-bar__icon"
+       >
+      <label class="find-form__label">
+        <input
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          name="search"
+          data-qa="keypress"
+          class="search-bar__input"
+        >
+      </label>
+    </form>
```

Comment:
```suggestion
    <form 
      action="/search" 
      method="GET" 
      data-qa="big" 
      class="search-bar"
    >
      <img 
        src="./images/Search.svg" 
        alt="Search icon" 
        class="search-bar__icon" 
      />

      <label class="find-form__label">
        <input 
          type="text" 
          placeholder='Try "Los Angeles"' 
          name="search" 
          data-qa="keypress" 
          class="search-bar__input"
        />
      </label>
    </form>
```

Reply:
pay attention to your indents on this form and on below!

Reply:
sry indents in css or in index.html code style?